docs/design/kata-guest-image-management-design.md

```diff
@@ -123,4 +168,4 @@ nerdctl run --runtime io.containerd.kata.v2 --snapshotter nydus --label io.kuber
 References:
 [1] [[RFC] Image management proposal for hosting sharing and peer pods](https://github.com/confidential-containers/confidential-containers/issues/137)
 [2] https://github.com/containerd/containerd/blob/main/docs/content-flow.md
-
+[3] [Move guest pull ability to a configurable component](https://github.com/kata-containers/kata-containers/issues/9266)
```

Comment:
While you're at it, you could improve the references section by making it a numbered list:

```suggestion
1. [[RFC] Image management proposal for hosting sharing and peer pods](https://github.com/confidential-containers/confidential-containers/issues/137)
2. https://github.com/containerd/containerd/blob/main/docs/content-flow.md
3. [Move guest pull ability to a configurable component](https://github.com/kata-containers/kata-containers/issues/9266)
```